backend/tracim_backend/models/data.py

```diff
@@ -724,8 +728,19 @@ def __setattr__(self, key: str, value: 'mixed'):
 
         super().__setattr__(key, value)
 
+    @property
+    def is_active(self) -> bool:
+        return not self.is_deleted and not self.is_archived
+
+    @property
+    def is_readonly(self) -> bool:
+        return False
```

Comment:
Why hardcoded False ?

Reply:
because is_readonly is a future feature distinct from "not editable" , a content with is_readonly as true (this will probably need something in database) will be always not editable.